source/_components/notify.nfandroidtv.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "Notifications for Android TV / FireTV"
+description: "Instructions how to add user notifications to Home Assistant."
+date: 2016-10-21 13:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nfandroidtv.png
+ha_category: Notifications
+ha_release: 0.31
+---
+
+
+Notification platform for [Notifications for Android TV](https://play.google.com/store/apps/details?id=de.cyberdream.androidtv.notifications.google&hl=de) and [Notifications for FireTV](https://play.google.com/store/apps/details?id=de.cyberdream.firenotifications.google).
+The notifications are in the global scope of your Android TV device. They will be displayed regardless of which application is running.
+The In-App purchases only apply to the client for Android smartphones, so there isn't any limit when pushing notifications from HASS.
+
+To enable the notification platform, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
```

Comment:
This example would preferably be minimized.
It's optional to have a fuller example also with recommend settings like this.